src/language/dafnyInstallation.ts

```diff
@@ -84,9 +84,9 @@ export async function getCompilerRuntimePath(context: ExtensionContext): Promise
 }
 
 export async function getLanguageServerRuntimePath(context: ExtensionContext): Promise<string> {
-  const configuredPath = ifNullOrEmpty(
+  const configuredPath = await ifNullOrEmpty(
     getConfiguredLanguageServerRuntimePath(),
-    LanguageServerConstants.GetDefaultPath(await getConfiguredVersion())
+    async () => LanguageServerConstants.GetDefaultPath(await getConfiguredVersion())
```

Comment:
Would `getConfiguredLanguageServerRuntimePath() || LanguageServerConstants.GetDefaultPath(await getConfiguredVersion())` work?

Reply:
(Same for the other function, this `ifnullorempty` that takes a lazy promise is a bit intimidating